src/include/planner/create_plan.h

```diff
@@ -33,6 +34,11 @@ class CreatePlan : public AbstractPlan {
   CreatePlan &operator=(const CreatePlan &) = delete;
   CreatePlan(CreatePlan &&) = delete;
   CreatePlan &operator=(CreatePlan &&) = delete;
+  ~CreatePlan() {
+    if (trigger_when) {
+      delete trigger_when;
```

Comment:
It is observed that in `src/include/parser/create_statement.h` you also have 
```
if (trigger_when) {
      delete trigger_when;
}
```
Then would it be possible that `~CreatePlan` has double free problem?